_layouts/onsite-groups/location.html

```diff
@@ -19,15 +19,15 @@ <h1 class="font-family-condensed-extra text-uppercase">{{ page.location.title }}
           {% endfor %}
         </div>
         <div>
-          <a class="text-white underline" href="/groups/onsite">See other group locations</a>
+          <a class="text-white underline" href="/groups/site-based#locations">See other group locations</a>
```

Comment:
This doesn't seem to link to anything. Was the intention to link down the page to locations or something else?

Reply:
I can fix that